synapse/cortex.py

```diff
@@ -444,7 +447,7 @@ def _runCryoLoop(self):
                             layr.spliced.wait(timeout=1)
                             continue
 
-                        logger.warning('tanking splices: %d', len(items))
+                        logger.info('tanking splices: {:,}'.format(len(items)))
```

Comment:
Agreed, this should probably be
```
logger.info('tanking splices: %d', len(items))
```
To avoid the format string eval if log level is higher than info